spec/kong_tests.conf

```diff
@@ -8,6 +8,9 @@ cluster_listen_rpc = 127.0.0.1:9373
 ssl_cert = spec/fixtures/kong_spec.crt
 ssl_cert_key = spec/fixtures/kong_spec.key
 
+admin_ssl_cert = spec/fixtures/kong_spec.crt
+admin_ssl_cert_key = spec/fixtures/kong_spec.key
```

Comment:
Why complicating our lives in tests? If anything, the admin API should only be available on SSL since it's being used to provision security credentials.